app/controllers/concerns/hyacinth/digital_objects/captions.rb

```diff
@@ -38,7 +69,11 @@ def clear_captions_and_reimport_transcript
 
     # Never trust parameters from the scary internet, only allow the white list through.
     def captions_params
-      params.permit(:captions_text)
+      params.permit(:captions_vtt, :file)
+    end
+
+    def synchronized_transcript_params
+      params.permit(:synchronized_transcript_text)
```

Comment:
Do we need to permit `:file` here?

Reply:
We don't have the corresponding form object/ability to edit as text for the synchronized transcript - only what comes from the synchronizer.

Reply:
Ah right!  In that case, never mind!  Approving the PR now. 👍